Audit/sp-operate-lexluthr (#259)
filecoin-project/lotus-docs: content/en/kb/_index.md

---
title: "Knowledge Base"
description: "Knowledge Base for lotus distribution of filecoin"

> lead: "Knowledge Base for lotus distribution of filecoin. The entire knowledge base is divided into articles and solutions. Articles are informative and solutions deal with specific errors."

Reviewer: ```suggestion
lead: "The knowledge base for Lotus. This knowledge base is divided into articles and solutions. Articles are informative, while solutions deal with specific errors."
```
